docs/GetStarted/buildRPC.md

```diff
@@ -107,16 +124,17 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
           129 |  ILibSpawnNormalThread((voidfp)(&LME_Thread), module);
           ...
         ```
+2. Copy to the executable to a managed device. 
 
-2. Confirm a successful build:
+3. Confirm a successful build:
 
     === "Linux"
         ``` bash
         sudo ./rpc version
         ```
     === "Windows"
         ``` bash
-        .\rpc version
+        rpc version
```

Comment:
We need the '.\\' on all Windows cmds, while Command Prompt doesn't care one way or the other, it is for Powershell.  This allows the copy/paste to work across both.